invenio_rdm_migrator/streams/communities/models.py

```diff
@@ -25,6 +25,17 @@ class Community:
     _table_name: InitVar[str] = "communities_metadata"
 
 
+@dataclass
+class RDMParentCommunityMetadata:
```

Comment:
moved here since it is semtically correct and it needs to be reused in two places (records and requests)